docs/install/iso-install.md

```diff
@@ -17,6 +17,10 @@ During the installation you can either choose to form a new cluster, or join the
 
 Note: This [video](https://youtu.be/97ADieBX6bE) shows a quick overview of the ISO installation.
 
+<div class="text-center">
+<iframe width="950" height="475" src="https://www.youtube.com/embed/97ADieBX6bE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+
```

Comment:
add embedded Youtube videos in 'iso installation' page. this video is the same as in the intro page.